src/SapAct/Services/SQLService.cs

```diff
@@ -284,7 +282,21 @@ private static string EmitTableUpdateCommand(string tableName, IEnumerable<strin
 	{
 		StringBuilder tableUpdateSB = new();
 
-		List<string> addedColumns = schemaDescriptor.Columns.Select(x => x.ColumnName).Except(columns).ToList(); //find added columns - only additive schema changes are applied
+		var schemaDescriptorColumnNames = schemaDescriptor.Columns.Select(x => x.ColumnName).ToList();
+
+		List<string> addedColumns = schemaDescriptorColumnNames
+			.Except(columns, StringComparer.OrdinalIgnoreCase).ToList(); //find added columns - only additive schema changes are applied - ignore casing
```

Comment:
we call the `.ToList()` in both lines, so we need to call it on the first?

Reply:
the second one is "different" list (note the except)

Reply:
but the point was, if you don't use `schemaDescriptorColumnNames` ever, I didn't see usage, you don't need to call it.. maybe I missed it.

Reply:
it is used --twice

Reply:
had a call later, understood what Ricard was trying to raise, none of those have to "lists" so addressed now